### PR TITLE
Query update

### PR DESCRIPTION
**Pull request by Draxeytina**
- Update animals table.
  - Insert data
- Update data.sql, schema.sql and query.sql files.
- Delete all animals born after Jan 1st, 2022.
- Update all animals' weights that are negative to be their weight multiplied by -1.
- Write queries to answer the following questions:
  - How many animals are there?
  - How many animals have never tried to escape?
  - What is the average weight of animals?
  - Who escapes the most, neutered or not neutered animals?
  - What is the minimum and maximum weight of each type of animal?
  - What is the average number of escape attempts per animal type of those born between 1990 and 2000?
  - 
query_screenshot3

query_screenshot4

query_screenshot5

query_screenshot6

query_screenshot7

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -3,4 +3,33 @@
...
+SELECT * FROM animals;
+
+vet_clinic=# BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT deleteYoung;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO deleteYoung;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+SELECT * FROM animals;
```
Comment (ShahierNashaat) [inline data.sql]: - [ ] Kindly be noticed that the transaction commands should be in the `queries.sql` file.

File: queries.sql [hunk 2]
```diff
@@ -29,4 +29,13 @@ SELECT * from animals WHERE name NOT IN ('Gabumon');
...
-SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- ################################# DAY 2 ######################################
+
+SELECT count(id) FROM animals;
+SELECT count(id) FROM animals WHERE escape_attempts = 0;
+SELECT avg(weight_kg) from animals;
+SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
```
Comment (ShahierNashaat) [inline queries.sql]: - [ ] Kindly be noticed that the question says `who escapes the most, neutered or not neutered animals?` not `who has the most escapes, neutered or not neutered?`.
```sql
SELECT neutered, MAX(escape_attempts) FROM animals
  GROUP BY neutered;
```

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @Draxeytina, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 2 above)
Comment (Draxeytina) [reply queries.sql]: Oh now I understand thank you. I had misinterpreted that.

File: data.sql (hunk 1 above)
Comment (Draxeytina) [reply data.sql]: Noted thank you!

**Review by Deepakdanger — APPROVED** [review]
Hi @Draxeytina, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**